src/utils/query.js

```diff
@@ -791,7 +792,7 @@ export function dispatchListenerResponse({
       ? docData.docChanges()
       : docData.docChanges;
   // Dispatch different actions for doc changes (only update doc(s) by key)
-  if (docChanges && docChanges.length < docData.size) {
+  if (docChanges && docChanges.length < Math.min(MAX_MULTI_ACTION, docData.size)) {
```

Comment:
Where does this max of 15 come from? We should note this in docs and make it an option if it isn't an amount enforced in firebase